modules/unsupported/tpk/src/main/java/org/geotools/tpk/TPKBundle.java

```diff
@@ -0,0 +1,173 @@
+package org.geotools.tpk;
```

Comment:
Copyright header missing: please add in all files.

Reply:
Ok copyright added now.